tree/20_cfe_basics/def.cf

```diff
@@ -20,4 +20,6 @@ bundle common ncf_def {
   vars:
       "package_module_query_installed_ifelapsed" string => "${node.properties[rudder][packages][installed_cache_expire]}";
       "package_module_query_updates_ifelapsed"   string => "${node.properties[rudder][packages][updates_cache_expire]}";
+  classes:
+      "mpf_stdlib_use_posix_utils" expression => "solaris";
```

Comment:
is this class used elsewhere is cfengine stdlib?

Reply:
This is only used in the cfengine paths lib to enable posix tools.